src/components/safe-messages/PaginatedMsgs/index.tsx

```diff
@@ -69,7 +69,7 @@ const PaginatedMsgs = (): ReactElement => {
   }
 
   return (
-    <Box mb={4} position="relative">
```

Comment:
Necessary?

Reply:
Leftover from before using the `key`. I've reverted it.